erpnext/projects/doctype/project/project.py

```diff
@@ -1382,6 +1384,12 @@ def get_item_groups_subtree(self, item_group):
 
 		return self._item_group_subtree[item_group]
 
+	def validate_notification(self, notification_type=None, child_doctype=None, child_name=None, throw=False):
+		if notification_type == "Ready to Close":
+			return self.ready_to_close and self.status == "To Close"
+		frappe.throw(_("Cannot send {0} notification because Repair Order status is not 'To Close'")
+					 .format(notification_type))
```

Comment:
this is problematic: will not throw for notification_type == "Ready to Close", will throw for all other notification types. fix logic